internal/extproc/translator/openai_gcpgemini.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Envoy AI Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+// Copyright Envoy AI Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package translator
+
+import (
+	"io"
+
+	extprocv3 "github.com/envoyproxy/go-control-plane/envoy/service/ext_proc/v3"
+	"google.golang.org/genai"
+
+	"github.com/envoyproxy/ai-gateway/internal/apischema/openai"
+)
+
+type GenerateContentRequest struct {
```

Comment:
GCP type should be moved under apischema
https://github.com/envoyproxy/ai-gateway/tree/main/internal/apischema